examples/Thermostat/slice/Reading.slice

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) ZeroC, Inc.
+
+module Igloo
+
+/// Represents a snapshot of the thermostat state at a point in time.
+struct Reading {
+    /// The point in time where this Reading was taken.
```

Comment:
```suggestion
    /// The point in time when this Reading was taken.
```
Since we're talking about a point in time (when), not a point in space (where).